flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/common/config/PulsarClientFactory.java

```diff
@@ -153,8 +153,8 @@ public static PulsarClient createClient(PulsarConfiguration configuration) {
     }
 
     /**
-     * PulsarAdmin shares almost the same configuration with PulsarClient, we use this
-     * creating method for creating PulsarAdmin instead of using {@link ClientConfigurationData}.
+     * PulsarAdmin shares almost the same configuration with PulsarClient, we use this creating
+     * method for creating PulsarAdmin instead of using {@link ClientConfigurationData}.
```

Comment:
This is unrelated to my change, but the CI complains about this

Reply:
This is being worked on via https://github.com/apache/flink/pull/20154

Reply:
@qingwei91 If you rebase, the issue will be resolved